packages/ecs-agent/0002-bottlerocket-remove-unsupported-capabilities.patch

```diff
@@ -1,37 +1,35 @@
-From b84708cfb634e84ca0ef2ac95292ddc1b370b41f Mon Sep 17 00:00:00 2001
-From: Sean McGinnis <stmcg@amazon.com>
-Date: Tue, 21 Feb 2023 19:06:43 +0000
+From df30b1e65ef7bb7d8ad8903fd75f196de8e9f46b Mon Sep 17 00:00:00 2001
+From: Shikha Vyaghra <vyaghras@amazon.com>
+Date: Wed, 23 Aug 2023 22:16:58 +0000
 Subject: [PATCH] bottlerocket: remove unsupported capabilities
 
-Signed-off-by: Sean McGinnis <stmcg@amazon.com>
-Signed-off-by: Arnaldo Garcia Rincon <agarrcia@amazon.com>
 ---
  agent/app/agent_capability.go | 8 ++++----
  1 file changed, 4 insertions(+), 4 deletions(-)
 
 diff --git a/agent/app/agent_capability.go b/agent/app/agent_capability.go
-index 0f78c00..746d87a 100644
+index e02e612..a0f6767 100644
```

Comment:
From what I can tell, this is simply line changes for the patch. I would rather you keep attribution to the original authors in the file then.

Reply:
Sure, I will update the commit.